resources/sdk/clisdkclient/scripts/preprocess-swagger.js

```diff
@@ -1,175 +1,186 @@
+		if (!keys.includes('_')) duplicateCommandMappings.delete(keys);
+	});
 
-	return duplicateCommandMappings
+	return duplicateCommandMappings;
 }
 
 function initialProcessOfDefinitions(newSwagger, resourceDefinitions) {
-	let superCommands = new Set()
-	let includedSwaggerPathObjects = {}
-	for (const path of Object.keys(newSwagger["paths"])) {
+	let superCommands = new Set();
+	let includedSwaggerPathObjects = {};
+	for (const path of Object.keys(newSwagger['paths'])) {
 		if (Object.keys(resourceDefinitions).includes(path)) {
-			includedSwaggerPathObjects[path] = newSwagger["paths"][path]
+			includedSwaggerPathObjects[path] = newSwagger['paths'][path];
 
 			for (let value of Object.values(includedSwaggerPathObjects[path])) {
-				let commandName = resourceDefinitions[path].name || value.tags[0]
-				commandName = commandName.toLowerCase().replace(" ", "")
-				const supercommand = resourceDefinitions[path].supercommand
-				if (!supercommand)
-					superCommands.add(commandName)
-				else
-					superCommands.add(supercommand.toLowerCase())
+				let commandName = resourceDefinitions[path].name || value.tags[0];
+				commandName = commandName.toLowerCase().replace(' ', '');
+				const supercommandList = resourceDefinitions[path].supercommand;
+				if (!supercommandList) superCommands.add(commandName);
+				else {
+					const supercommandArray = supercommandList.split('.');
+					superCommands.add(supercommandArray[0].toLowerCase());
+				}
 			}
 		}
 	}
 
-	return [superCommands, includedSwaggerPathObjects]
+	return [superCommands, includedSwaggerPathObjects];
 }
 
 function retrieveSwagger(newSwaggerPath) {
-	let newSwagger
+	let newSwagger;
 	// Retrieve new swagger
 	if (fs.existsSync(newSwaggerPath)) {
-		console.log(`Loading new swagger from disk: ${newSwaggerPath}`)
-		newSwagger = JSON.parse(fs.readFileSync(newSwaggerPath, 'utf8'))
+		console.log(`Loading new swagger from disk: ${newSwaggerPath}`);
+		newSwagger = JSON.parse(fs.readFileSync(newSwaggerPath, 'utf8'));
 	} else if (newSwaggerPath.toLowerCase().startsWith('http')) {
-		console.log(`Downloading new swagger from: ${newSwaggerPath}`)
-		newSwagger = JSON.parse(downloadFile(newSwaggerPath))
+		console.log(`Downloading new swagger from: ${newSwaggerPath}`);
+		newSwagger = JSON.parse(downloadFile(newSwaggerPath));
 	} else {
-		throw `Invalid newSwaggerPath: ${newSwaggerPath}`
+		throw `Invalid newSwaggerPath: ${newSwaggerPath}`;
 	}
 
-	return newSwagger
+	return newSwagger;
 }
 
 function downloadFile(url) {
-	var i = 0
+	var i = 0;
 	while (i < 10) {
-		i++
-		console.log(`Downloading file: ${url}`)
+		i++;
+		console.log(`Downloading file: ${url}`);
 		// Source: https://www.npmjs.com/package/download-file-sync
-		var file = childProcess.execFileSync('curl', ['--silent', '-L', url], { encoding: 'utf8', maxBuffer: 1024 * 1024 * 1024 })
+		var file = childProcess.execFileSync('curl', ['--silent', '-L', url], { encoding: 'utf8', maxBuffer: 1024 * 1024 * 1024 });
```

Comment:
Define the max buffer as a constant here.  Otherwise someone reading the code is going to have to do the math to figure out the buffer size.

Reply:
Fixed. Created a constant number at the very top of the file and reference it here.